sdk/clientcore/core/src/main/java/module-info.java

```diff
@@ -22,5 +22,9 @@
     exports io.clientcore.core.util.serializer;
     exports io.clientcore.core.util.auth;
 
+    // This is for exporting IOUtils and other utils classes
+    exports io.clientcore.core.implementation.util;
+    exports io.clientcore.core.implementation;
```

Comment:
If we really need to export these we should export them to specific modules, otherwise we may as well just make the code here non-implementation as this effectively makes it public API

Reply:
Rolling back these changes for now. I'll keep them in the feature/azure-core-v2 branch since that's where we need them at the moment.